account_analytic_tag/models/account_analytic_tag.py

```diff
@@ -0,0 +1,17 @@
+from odoo import fields, models
+
+
+class AccountAnalyticTag(models.Model):
+    _name = "account.analytic.tag"
+    _description = "Analytic Tags"
+
+    name = fields.Char(string="Analytic Tag", index=True, required=True)
+    color = fields.Integer("Color Index")
+    active = fields.Boolean(
+        default=True,
+        help="Set active to false to hide the Analytic Tag without removing it.",
+    )
+    company_id = fields.Many2one("res.company", string="Company")
+    account_analytic_id = fields.Many2one(
+        "account.analytic.account", string="Analytic Account", required=True
+    )
```

Comment:
This m2o field seems wrong ?

Reply:
Can you add a help text that explains the purpose of this `account_analytic_id` field? Perhaps a more specific label than `Analytic Account` would help too.